src/Client/StorePaymentMethodLightningNetwork.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+use BTCPayServer\Http\CurlClient;
+
+/**
+ * Handles a stores LightningNetwork payment methods.
+ *
+ * @see https://docs.btcpayserver.org/API/Greenfield/v1/#tag/Store-Payment-Methods-(Lightning-Network)
+ */
+class StorePaymentMethodLightningNetwork extends AbstractStorePaymentMethodClient
+{
+    public function getPaymentMethods(string $storeId): array
+    {
+        $url = $this->getApiUrl() . 'stores/' . urlencode($storeId) . '/payment-methods/LightningNetwork';
```

Comment:
Should we put `LightningNetwork` in a constant? It seams to be used 4+ times in this file? Does that make sense?

Reply:
Yes good point, we can use that. We could also use it then for the api endpoints segments but maybe that is a bit too much as it might be too confusing then.

Reply:
Why would it be confusing? A constant is not hard to grasp, so I guess it would be fine, no? I'll let you be the judge on this.